### PR TITLE
Convert sensors to css grid layout

### PR DESCRIPTION
**Pull request by andrewtran30**
- Converted IonRows and IonCol to a 3x2 CSS Grid layout
- Added new styles to the SensorsTab CSS module
- **KEPT** the IonCard (subject to change) and stored it inside grid cells
- Removed unused dependencies in SensorsTab

### CODE REVIEW

File: src/Tablet/Sensors/SensorsTab.module.css [hunk 1]
```diff
@@ -1,3 +1,16 @@
...
+  grid-template-columns: auto auto auto;
+  grid-template-rows: auto auto;
+  grid-gap: 0px;
+  padding-right: 5%;
+  padding-left: 5%;
+}
+
+.gridItem {
+  background-color: #1b1b1b;
+  border: 4px outset white;
```
Comment (grgur) [inline src/Tablet/Sensors/SensorsTab.module.css]: It looks really ugly with this border 😆 

image

File: src/Tablet/Sensors/SensorsTab.js [hunk 2]
```diff
@@ -99,28 +99,22 @@ const SensorsTab = () => {
...
-        {renderCard(types[col++])}
-        {renderCard(types[col++])}
-      </IonRow>
-      <IonRow>
-        {renderCard(types[col++])}
-        {renderCard(types[col++])}
-        {renderCard(types[col++])}
-      </IonRow>
+    <div className={s.gridContainer}>
+      <div className={s.gridItem}>{renderCard(types[col++])}</div>
```
Comment (grgur) [inline src/Tablet/Sensors/SensorsTab.js]: `div.gridItem` looks like an overkill. Removing this element altogether should do
image

Comment (grgur) [reply src/Tablet/Sensors/SensorsTab.js]: In that casse all you need is this simple loop
```jsx
<div className={s.gridContainer}>
  {types.map(renderCard)}
</div>
```

File: src/Tablet/Sensors/SensorsTab.module.css [hunk 3]
```diff
@@ -1,3 +1,16 @@
-.column {
+.gridContainer {
   margin-top: 20px;
+  display: grid;
+  grid-template-columns: auto auto auto;
```
Comment (grgur) [inline src/Tablet/Sensors/SensorsTab.module.css]: `grid-template-columns: 1fr 1fr 1fr` looks much better to m e as the columns will be equal in size

image

Comment (andrewtran30) [reply src/Tablet/Sensors/SensorsTab.module.css]: i second that

File: src/Tablet/Sensors/SensorsTab.js (hunk 2 above)
Comment (andrewtran30) [reply src/Tablet/Sensors/SensorsTab.js]: cant believe i didnt recognize this!

**Pull request closed without merging**